### DISCUSSION TITLE
How do I find out if our Institution is already a member, and further, how would I find out who the point of contact is?

### DISCUSSION THREAD

**Discussion by rmleonard** [community/community#140356] [GitHub Education]
### Hi everyone,

Greetings!

How do I find out if our Institution is already a member, 
and further, 
how would I find out who the point of contact is?

**Comment by davevad93** [comment]
Hi @rmleonard ,

One way to find out is by applying [here](https://education.github.com/discount_requests/application) for the `Student Developer Pack` as a student using your academic email, if your school isn't listed on the application form, that probably means they aren't part of the GitHub Campus program. To be sure, you should ask directly to your academic institution if they are partners of [GitHub Campus](https://education.github.com/schools) or [Microsoft Education Program](https://www.microsoft.com/en-in/education). 

You can also contact Education support by opening a ticket [here](https://support.github.com/contact/education), although there doesn't seem to be an appropriate category for this kind of question.

**Comment by rmleonard** [comment]
I've been trying to complete the Education application - it keeps insisting that I am not on campus, but rather "very far away" - at first - all it wanted from "me" was proof of employment, that showed a date, the name of the institution - now, it seems to keep asking for more and different things - does it "remember" any of what I did in prior connections ? some of my students think I should start making faces, and act goofy, so we did some of those shots... didn't help at all... 

so, 
A) How do find out what it "really" needs to complete my application (which could still be moot, as I cannot find ANYWHERE, any list of approved Universities) ...
B) I work at Cal Poly Pomona, in Pomona, California, USA - How do I find out if the University is approved, and who our point of contact is?

**Reply by davevad93** [reply]
@samus-aran , is there any list available or any way to know if an academic institution is already a GitHub Campus partner?
